### PR TITLE
Added dev tools testing info

### PR DESCRIPTION
**Pull request by Abhi-M**
This PR closes OWASP/wstg#493.

- [ ] This PR handles the issue and requires no additional PRs.
- [x] You have validated the need for this change.

**What did this PR accomplish?**

- Readers now would be able to use browser dev tools for performing some basic web app security tests

### CODE REVIEW

**Comment by victoriadrake** [comment]
Hi @Abhi-M!

I think what we meant in #493 by "add to the appendix" was that you would contribute new content, perhaps as a new Appendix F section, with the information about testing with browser dev tools. Is this something you're willing to undertake?

Simply adding the link is not as valuable, as few people will end up seeing it. A new appendix section, with detailed instructions and links to existing chapters for more information, would be much more helpful to WSTG users. Let us know if you'd like to champion this!

**Comment by Abhi-M** [comment]
Oops, my bad. I will be more than happy to take that up. Thanks for the clarification.

**Comment by victoriadrake** [comment]
No worries! Thanks for taking it on!

**Comment by kingthorin** [comment]
@Abhi-M are you still able to address this?

**Comment by Abhi-M** [comment]
Not at the moment. I'm going through a slightly busy schedule. Please free to close/re-assign.

**Comment by kingthorin** [comment]
I'll tackle it.

**Comment by ThunderSon** [comment]
@kingthorin I am around, I just believe there's still some content missing? When you require my review, let me know. (Will most probably be done on Sunday when requested).

**Comment by kingthorin** [comment]
Thanks, yes this is very much still a work in progress. 

On top of assembling all sections similarly to what's already here I plan to add a 'use cases' or 'related testing' section/list to each level 2. Plus I need to review and potentially adapt the BHIS stuff linked in the original issue.

**Comment by kingthorin** [comment]
You're of course welcome to review this, but please keep in mind it is far from finished.

**Comment by ThunderSon** [comment]
No rush, just keeping this in my tasks. No need to review so soon :)

**Comment by ThunderSon** [comment]
@kingthorin hello good sir. Update me on the status of this PR so we can tackle it together.

**Comment by kingthorin** [comment]
I just need to get back to adding content 👍

**Comment by kingthorin** [comment]
I believe part one of this is now complete. Tomorrow/this week I'll tackle:
- Use cases/scenarios for each covered piece of functionality (H2).
- Watching the BHIS video and adding further suggestions/tidbits.

**Comment by kingthorin** [comment]
This one is getting close. I just need to watch the BHIS video and see if there are any tips/tricks worth extracting/adding.

**Comment by kingthorin** [comment]
Add ref: https://github.com/gregmalcolm/wacky-wandas-wicked-weapons-frontend/blob/fix-it/README.md (bit.ly/wickedweapons)

File: document/6-Appendix/F-Leveraging_Dev_Tools.md [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+### Google Chrome
+
+1. Click the `Application` tab.
+2. Expand `Local Storage` under the `Storage` heading.
+3. Select the relevant domain name.
+4. Double click in the `Value` column to edit any cookie value.
+5. Double click in the applicable Cell to edit the `Key` or `Value`.
+
+> Note: Editing `Session Storage` or `Index DB` follows essentially the same steps.
+> Note: Items can be added or deleted via the right-click context menu.
```
Comment (ThunderSon) [inline document/6-Appendix/F-Leveraging_Dev_Tools.md:130]:
```suggestion
> Note: Editing `Session Storage` or `Index DB` follows essentially the same steps.
>
> Note: Items can be added or deleted via the right-click context menu.
```

**Review by ThunderSon — APPROVED** [review]

**Pull request merged**